src/instructlab/sdg/generate_data.py

```diff
@@ -183,6 +183,7 @@ def _context_init(
     save_freq: int,
     batch_num_workers: Optional[int],
     batch_size: Optional[int],
+    max_num_tokens: Optional[int] = 4096,
```

Comment:
Could we reuse the constant defined in llmblock.py for this instead of hardcoding 4096 here and elsewhere? All these files are already importing things from llmblock anyway.